core/src/mindustry/entities/comp/HealthComp.java

```diff
@@ -34,7 +34,6 @@ void killed(){
     void kill(){
         if(dead) return;
 
-        health = 0;
```

Comment:
Why was this changed?

Reply:
ye

Reply:
Dunno, might of been removed when I was trying to debug why the wrecks weren't dying

Reply:
Even if it's related to wrecks, it should at most be changed to `Math.min(health, 0)`. A killed unit should have 0 health.